lib/gooddata/lcm/lcm2.rb

```diff
@@ -101,6 +101,7 @@ def respond_to_missing?(name, *_args)
 
       provision: [
         EnsureReleaseTable,
+        CollectDataProduct,
```

Comment:
We should use this also in the release brick. It is needed e.g. in `CreateSegmentMasters` action.